Assets/Scripts/BodyController.cs

```diff
@@ -10,10 +10,19 @@ public class BodyController : MonoBehaviour
     // private Rigidbody rb;
 
     void Start() {
+		
     }
 
     void FixedUpdate() {
-		transform.Translate(0f, -(movementSpeed * Input.GetAxis("Vertical2") * Time.deltaTime), 0f);
-		transform.Rotate(0f, 0f, rotationSpeed * Input.GetAxis("Horizontal2") * Time.deltaTime);
+		string[] names = Input.GetJoystickNames();
+		float moveHorizontal = Input.GetAxis("Vertical2");
+		float moveVertical = Input.GetAxis("Horizontal2");
```

Comment:
Do you have the names backwards here? Should this be `float moveHorizontal = Input.GetAxis("Horizontal2");``float moveVertical = Input.GetAxis("Vertical2");`? It may explain the perceived mixup between `s` and `w` in `InputManager` below